### DISCUSSION TITLE
[Support]: [MQTT] HA Notifications ghost events

### DISCUSSION THREAD

**Discussion by Morphy99** [blakeblackshear/frigate#12497] [Config Support]
### Describe the problem you are having

I seem to occasionally get notifications for a Person object, however, the event doesn't exist and the snapshot doesn't show a box for an object anywhere. It also fails to download after a few minutes despite showing up in the notification preview on my mobile device.

### Version

0.14-beta4

### Frigate config file

```yaml
mqtt:
  host: core-mosquitto
  user: mqtt
  password: pass

detectors:
  coral0:
    type: edgetpu
    device: pci:0

auth:
  enabled: False

objects:
  track:
    - person
    - car
    - dog
    - bird
    - mouse
  filters:
    person:
      threshold: 0.7
      max_ratio: 0.6

record:
  expire_interval: 10
  enabled: true
  retain:
    days: 2
    mode: all
  events:
    retain:
      default: 20
      mode: motion
    pre_capture: 2
    post_capture: 30

snapshots:
  enabled: true
  timestamp: false
  bounding_box: true
  crop: false
  retain:
    default: 20

detect:
  stationary:
    interval: 10

motion:
  improve_contrast: true

ffmpeg:
  hwaccel_args: preset-vaapi
  output_args:
    record: preset-record-generic-audio-copy

go2rtc:
  streams:
    path:
      - "ffmpeg:http://xxx.xxx.xxx.xxx/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=user&password=pass#video=copy#audio=copy#audio=opus"
    path_sub:
      - "ffmpeg:http://xxx.xxx.xxx.xxx/flv?port=1935&app=bcs&stream=channel0_sub.bcs&user=user&password=pass"

birdseye:
  enabled: false

cameras:
  path:
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/path
          input_args: preset-rtsp-restream
          roles:
            - record
            - detect
    detect:
      width: 2048
      height: 1536
      fps: 8
    mqtt:
      required_zones:
        - path_close
        - path_far
        - path_field
    snapshots:
      required_zones:
        - path_close
        - path_far
        - path_field
    objects:
      track:
        - person
        - dog
        - bird
        - mouse
      filters:
        person:
          mask:
            - 1231,390,1178,651,964,612,955,371
            - 952,83,886,84,879,145,952,149
            - 1183,0,949,0,915,387,1148,393
        bird:
          mask:
            - 1451,659,1228,257,1314,233,1545,588,2048,1212,2048,1364
    motion:
      mask:
        - 751,0,0,0,0,71,751,69
        - 1255,325,1442,0,2048,0,2048,1536
        - 0,888,916,133,637,139,0,443
    zones:
      path_close:
        coordinates: 0,1536,2048,1536,2048,612,337,590,0,889
        filters:
          person:
            min_area: 100000
            max_area: 200000
      path_far:
        coordinates: 871,108,2048,0,2048,612,0,614
        filters:
          person:
            min_area: 10000
            max_area: 70000
          bird:
            max_area: 10000
      path_field:
        coordinates: 871,108,752,57,0,138,0,614
        filters:
          person:
            max_area: 20000
    review:
      alerts:
        required_zones:
          - path_close
          - path_far
          - path_field
```

### Relevant log output

```shell
Home Assistant automation:
id: Camera_Path_Person_Alert
alias: Camera Path Person Alert
mode: restart
trace:
  stored_traces: 20
trigger:
  platform: mqtt
  topic: frigate/events
  payload: new
  value_template: '{{ value_json.type }}'
condition:
  - condition: state
    entity_id: input_boolean.camera_path_notifications
    state: 'on'
  - condition:
      - '{{ trigger.payload_json[''after''][''label''] == ''person'' }}'
      - '{{ ''path'' in trigger.payload_json[''after''][''camera''] }}'
  - condition: template
    value_template: >-
      {{ (as_timestamp(now()) -
      as_timestamp(state_attr('automation.camera_path_person_alert',
      'last_triggered') | default(0), 0) | int > 30)}}
action:
  - service: notify.mobile_app
    data_template:
      message: A person was detected on the Path camera.
      title: Camera Alert
      data:
        channel: Cameras Max
        ttl: 0
        priority: high
        media_stream: alarm_stream_max
        image: >-
          https://nabuurl.ui.nabu.casa/api/frigate/notifications/{{trigger.payload_json["after"]["id"]}}/thumbnail.jpg?format=android
        tag: '{{trigger.payload_json["after"]["id"]}}'
        when: '{{trigger.payload_json["after"]["start_time"]|int}}'
        actions:
          - action: URI
            title: View Snapshot
            uri: >-
              https://nabuurl.ui.nabu.casa/api/frigate/notifications/{{trigger.payload_json["after"]["id"]}}/snapshot.jpg
          - action: MUTE_PATH
            title: Mute
        clickAction: /lovelace-security/path
  - service: input_text.set_value
    data_template:
      entity_id: input_text.cctv_alert_message
      value: A person was detected on the Path camera.
  - service: script.turn_on
    entity_id: script.cctv_alert_nest_lounge
  - service: script.turn_on
    entity_id: script.lights_on_away
  - service: script.turn_on
    entity_id: script.lights_on_home
  - service: notify.mobile_app
    data:
      message: TTS
      data:
        ttl: 0
        priority: high
        media_stream: alarm_stream_max
        tts_text: A person was detected on the Path camera.
  - wait_for_trigger:
      - platform: event
        event_type: mobile_app_notification_action
        event_data:
          action: MUTE_PATH
    timeout: '00:05:00'
    continue_on_timeout: false
  - service: input_boolean.turn_off
    entity_id: input_boolean.camera_path_notifications
```

### Frigate stats

```json
{
  "cameras": {
    "field": {
      "audio_dBFS": 0,
      "audio_rms": 0,
      "camera_fps": 6.1,
      "capture_pid": 448,
      "detection_enabled": true,
      "detection_fps": 0.1,
      "ffmpeg_pid": 454,
      "pid": 411,
      "process_fps": 6.1,
      "skipped_fps": 0
    },
    "front": {
      "audio_dBFS": 0,
      "audio_rms": 0,
      "camera_fps": 8.1,
      "capture_pid": 431,
      "detection_enabled": true,
      "detection_fps": 17.4,
      "ffmpeg_pid": 446,
      "pid": 409,
      "process_fps": 8.1,
      "skipped_fps": 0
    },
    "path": {
      "audio_dBFS": 0,
      "audio_rms": 0,
      "camera_fps": 8.1,
      "capture_pid": 438,
      "detection_enabled": true,
      "detection_fps": 4.6,
      "ffmpeg_pid": 452,
      "pid": 410,
      "process_fps": 8.1,
      "skipped_fps": 0
    },
    "stables": {
      "audio_dBFS": 0,
      "audio_rms": 0,
      "camera_fps": 7,
      "capture_pid": 455,
      "detection_enabled": true,
      "detection_fps": 0.7,
      "ffmpeg_pid": 468,
      "pid": 417,
      "process_fps": 7,
      "skipped_fps": 0
    }
  },
  "cpu_usages": {
    "1": {
      "cmdline": "/package/admin/s6/command/s6-svscan -d4 -- /run/service",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "116": {
      "cmdline": "bash ./run.user go2rtc-healthcheck",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "121": {
      "cmdline": "python3 -u -m frigate",
      "cpu": "5.0",
      "cpu_average": "5",
      "mem": "2.0"
    },
    "133": {
      "cmdline": "nginx: master process nginx",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "15": {
      "cmdline": "s6-supervise s6-linux-init-shutdownd",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "17": {
      "cmdline": "/package/admin/s6-linux-init/command/s6-linux-init-shutdownd -c /run/s6/basedir -g 3000 -C -B",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "171": {
      "cmdline": "nginx: worker process",
      "cpu": "0.1",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "172": {
      "cmdline": "nginx: worker process",
      "cpu": "0.1",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "173": {
      "cmdline": "nginx: worker process",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "174": {
      "cmdline": "nginx: worker process",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "175": {
      "cmdline": "nginx: cache manager process",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "24": {
      "cmdline": "s6-supervise certsync",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "25": {
      "cmdline": "s6-supervise nginx-log",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "26": {
      "cmdline": "s6-supervise s6rc-oneshot-runner",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "27": {
      "cmdline": "s6-supervise frigate-log",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "2777": {
      "cmdline": "/usr/bin/python3 -c from multiprocessing.resource_tracker import main;main(52)",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "28": {
      "cmdline": "s6-supervise go2rtc-healthcheck",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "29": {
      "cmdline": "s6-supervise frigate",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "30": {
      "cmdline": "s6-supervise nginx",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "308": {
      "cmdline": "bash ./run.user certsync",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "31": {
      "cmdline": "s6-supervise s6rc-fdholder",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "32": {
      "cmdline": "s6-supervise go2rtc-log",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "33": {
      "cmdline": "s6-supervise go2rtc",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "34": {
      "cmdline": "s6-supervise certsync-log",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "347": {
      "cmdline": "frigate.logger       ",
      "cpu": "0.1",
      "cpu_average": "0",
      "mem": "0.8"
    },
    "349": {
      "cmdline": "frigate.recording_manager",
      "cpu": "2.4",
      "cpu_average": "2",
      "mem": "1.2"
    },
    "356": {
      "cmdline": "frigate.review_segment_manager",
      "cpu": "1.0",
      "cpu_average": "0",
      "mem": "1.0"
    },
    "377": {
      "cmdline": "/usr/bin/python3 -c from multiprocessing.resource_tracker import main;main(63)",
      "cpu": "0.6",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "378": {
      "cmdline": "frigate.detector.coral0",
      "cpu": "5.4",
      "cpu_average": "11",
      "mem": "1.1"
    },
    "380": {
      "cmdline": "frigate.output       ",
      "cpu": "5.8",
      "cpu_average": "7",
      "mem": "1.0"
    },
    "409": {
      "cmdline": "frigate.process:front",
      "cpu": "15.7",
      "cpu_average": "25",
      "mem": "1.1"
    },
    "410": {
      "cmdline": "frigate.process:path ",
      "cpu": "4.0",
      "cpu_average": "6",
      "mem": "1.1"
    },
    "411": {
      "cmdline": "frigate.process:field",
      "cpu": "2.6",
      "cpu_average": "3",
      "mem": "1.0"
    },
    "417": {
      "cmdline": "frigate.process:stables",
      "cpu": "3.8",
      "cpu_average": "3",
      "mem": "1.0"
    },
    "431": {
      "cmdline": "frigate.capture:front",
      "cpu": "7.4",
      "cpu_average": "8",
      "mem": "1.0"
    },
    "438": {
      "cmdline": "frigate.capture:path ",
      "cpu": "7.4",
      "cpu_average": "8",
      "mem": "1.0"
    },
    "4407": {
      "cmdline": "sleep 60",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "4446": {
      "cmdline": "sleep 30s",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "446": {
      "cmdline": "ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel_flags allow_profile_mismatch -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format vaapi -user_agent FFmpeg Frigate/0.14.0-a4eb435 -rtsp_transport tcp -timeout 5000000 -i rtsp://127.0.0.1:8554/front -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac /tmp/cache/front@%Y%m%d%H%M%S%z.mp4 -r 8 -vf fps=8,scale_vaapi=w=2048:h=1536:format=nv12,hwdownload,format=nv12,format=yuv420p -threads 2 -f rawvideo -pix_fmt yuv420p pipe:",
      "cpu": "19.6",
      "cpu_average": "20",
      "mem": "0.6"
    },
    "448": {
      "cmdline": "frigate.capture:field",
      "cpu": "1.6",
      "cpu_average": "1",
      "mem": "1.0"
    },
    "45": {
      "cmdline": "/package/admin/s6/command/s6-ipcserverd -1 -- /package/admin/s6/command/s6-ipcserver-access -v0 -E -l0 -i data/rules -- /package/admin/s6/command/s6-sudod -t 30000 -- /package/admin/s6-rc/command/s6-rc-oneshot-run -l ../.. --",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "452": {
      "cmdline": "ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel_flags allow_profile_mismatch -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format vaapi -user_agent FFmpeg Frigate/0.14.0-a4eb435 -rtsp_transport tcp -timeout 5000000 -i rtsp://127.0.0.1:8554/path -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c copy /tmp/cache/path@%Y%m%d%H%M%S%z.mp4 -r 8 -vf fps=8,scale_vaapi=w=2048:h=1536:format=nv12,hwdownload,format=nv12,format=yuv420p -threads 2 -f rawvideo -pix_fmt yuv420p pipe:",
      "cpu": "14.3",
      "cpu_average": "15",
      "mem": "0.5"
    },
    "453": {
      "cmdline": "ffmpeg -hide_banner -v error -fflags nobuffer -flags low_delay -i http://xxx.xxx.xxx.21/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=*&password=* -c:v copy -c:a copy -user_agent ffmpeg/go2rtc -rtsp_transport tcp -f rtsp rtsp://127.0.0.1:8554/c5b3ea4e0afcfb67b2061f3d64e30f31",
      "cpu": "2.1",
      "cpu_average": "2",
      "mem": "0.2"
    },
    "454": {
      "cmdline": "ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel_flags allow_profile_mismatch -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format vaapi -user_agent FFmpeg Frigate/0.14.0-a4eb435 -rtsp_transport tcp -timeout 5000000 -i rtsp://127.0.0.1:8554/field_sub -r 6 -vf fps=6,scale_vaapi=w=1200:h=536:format=nv12,hwdownload,format=nv12,format=yuv420p -threads 2 -f rawvideo -pix_fmt yuv420p pipe:",
      "cpu": "6.2",
      "cpu_average": "6",
      "mem": "0.3"
    },
    "455": {
      "cmdline": "frigate.capture:stables",
      "cpu": "1.3",
      "cpu_average": "1",
      "mem": "0.9"
    },
    "46": {
      "cmdline": "/package/admin/s6-2.11.3.2/command/s6-fdholderd -1 -i data/rules",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "466": {
      "cmdline": "ffmpeg -hide_banner -v error -fflags nobuffer -flags low_delay -i http://xxx.xxx.xxx.26/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=*&password=* -map 0:v:0? -c:v:0 copy -map 0:a:0? -c:a:0 copy -map 0:a:0? -c:a:1 libopus -application:a:1 lowdelay -min_comp 0 -user_agent ffmpeg/go2rtc -rtsp_transport tcp -f rtsp rtsp://127.0.0.1:8554/14d49881e08e6e98d1e33eaa33b09d8e",
      "cpu": "3.1",
      "cpu_average": "2",
      "mem": "0.2"
    },
    "467": {
      "cmdline": "ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel_flags allow_profile_mismatch -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format vaapi -user_agent FFmpeg Frigate/0.14.0-a4eb435 -rtsp_transport tcp -timeout 5000000 -i rtsp://127.0.0.1:8554/field -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c copy /tmp/cache/field@%Y%m%d%H%M%S%z.mp4",
      "cpu": "1.0",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "468": {
      "cmdline": "ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel_flags allow_profile_mismatch -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format vaapi -user_agent FFmpeg Frigate/0.14.0-a4eb435 -rtsp_transport tcp -timeout 5000000 -i rtsp://127.0.0.1:8554/stables_sub -r 7 -vf fps=7,scale_vaapi=w=640:h=480:format=nv12,hwdownload,format=nv12,format=yuv420p -threads 2 -f rawvideo -pix_fmt yuv420p pipe:",
      "cpu": "2.9",
      "cpu_average": "2",
      "mem": "0.3"
    },
    "473": {
      "cmdline": "ffmpeg -hide_banner -v error -fflags nobuffer -flags low_delay -i http://xxx.xxx.xxx.28/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=*&password=* -c copy -user_agent ffmpeg/go2rtc -rtsp_transport tcp -f rtsp rtsp://127.0.0.1:8554/009a139c0286da10cc7874953fbc8883",
      "cpu": "0.8",
      "cpu_average": "0",
      "mem": "0.2"
    },
    "474": {
      "cmdline": "ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel_flags allow_profile_mismatch -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format vaapi -user_agent FFmpeg Frigate/0.14.0-a4eb435 -rtsp_transport tcp -timeout 5000000 -i rtsp://127.0.0.1:8554/stables -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c copy /tmp/cache/stables@%Y%m%d%H%M%S%z.mp4",
      "cpu": "0.6",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "475": {
      "cmdline": "ffmpeg -hide_banner -v error -fflags nobuffer -flags low_delay -i http://xxx.xxx.xxx.28/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=*&password=* -map 0:v:0? -c:v:0 copy -map 0:a:0? -c:a:0 copy -map 0:a:0? -c:a:1 libopus -application:a:1 lowdelay -min_comp 0 -user_agent ffmpeg/go2rtc -rtsp_transport tcp -f rtsp rtsp://127.0.0.1:8554/32ebec8e86c5b1068f401d89e59f9d24",
      "cpu": "2.9",
      "cpu_average": "2",
      "mem": "0.2"
    },
    "493": {
      "cmdline": "ffmpeg -threads 1 -f rawvideo -pix_fmt yuv420p -video_size 2048x1536 -i pipe: -threads 1 -f mpegts -s 960x720 -codec:v mpeg1video -q 8 -bf 0 pipe:",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "498": {
      "cmdline": "ffmpeg -threads 1 -f rawvideo -pix_fmt yuv420p -video_size 2048x1536 -i pipe: -threads 1 -f mpegts -s 960x720 -codec:v mpeg1video -q 8 -bf 0 pipe:",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "503": {
      "cmdline": "ffmpeg -threads 1 -f rawvideo -pix_fmt yuv420p -video_size 1200x536 -i pipe: -threads 1 -f mpegts -s 1611x720 -codec:v mpeg1video -q 8 -bf 0 pipe:",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "508": {
      "cmdline": "ffmpeg -threads 1 -f rawvideo -pix_fmt yuv420p -video_size 640x480 -i pipe: -threads 1 -f mpegts -s 960x720 -codec:v mpeg1video -q 8 -bf 0 pipe:",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.1"
    },
    "87": {
      "cmdline": "s6-log -b -- T 1 n0 s10000000 T /dev/shm/logs/nginx",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "88": {
      "cmdline": "s6-log -b -- T 1 n0 s10000000 T /dev/shm/logs/frigate",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "89": {
      "cmdline": "s6-log -b -- T 1 n0 s10000000 T /dev/shm/logs/go2rtc",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "90": {
      "cmdline": "s6-log -b -- T 1 n0 s10000000 T /dev/shm/logs/certsync",
      "cpu": "0.0",
      "cpu_average": "0",
      "mem": "0.0"
    },
    "98": {
      "cmdline": "/usr/local/go2rtc/bin/go2rtc -config=/dev/shm/go2rtc.yaml",
      "cpu": "6.8",
      "cpu_average": "6",
      "mem": "0.2"
    },
    "frigate.full_system": {
      "cpu": "35.0",
      "mem": "19.3"
    }
  },
  "detection_fps": 22.8,
  "detectors": {
    "coral0": {
      "detection_start": 0,
      "inference_speed": 7.66,
      "pid": 378
    }
  },
  "gpu_usages": {
    "intel-vaapi": {
      "gpu": "7.58%",
      "mem": "-%"
    }
  },
  "processes": {
    "go2rtc": {
      "pid": 98
    },
    "logger": {
      "pid": 347
    },
    "recording": {
      "pid": 349
    },
    "review_segment": {
      "pid": 356
    }
  },
  "service": {
    "last_updated": 1721295573,
    "latest_version": "0.13.2",
    "storage": {
      "/dev/shm": {
        "free": 7853,
        "mount_type": "tmpfs",
        "total": 7870.8,
        "used": 17.9
      },
      "/media/frigate/clips": {
        "free": 248297.3,
        "mount_type": "ext4",
        "total": 449581,
        "used": 201283.6
      },
      "/media/frigate/recordings": {
        "free": 248297.3,
        "mount_type": "ext4",
        "total": 449581,
        "used": 201283.6
      },
      "/tmp/cache": {
        "free": 7821,
        "mount_type": "tmpfs",
        "total": 7870.8,
        "used": 49.9
      }
    },
    "temperatures": {
      "apex_0": 50.8
    },
    "uptime": 1056,
    "version": "0.14.0-a4eb435"
  }
}
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

Coral

**Comment by hawkeye217** [comment]
This has been discussed often in past topics. `/events` is a raw MQTT topic, frigate has always published objects that were detected as true positives even if they did not qualify for an event to be created in the UI.

**Reply by NickM-27** [reply]
Also, the snapshot url doesn't show bounding boxes by default until the event is over. If you add `?bbox=1` to the end of the snapshot url it will show the bounding box

**Reply by Morphy99** [reply]
> This has been discussed often in past topics. `/events` is a raw MQTT topic, frigate has always published objects that were detected as true positives even if they did not qualify for an event to be created in the UI.

Sorry, I haven't searched the discussions as I've always just followed the docs. I will have a look for a better way, but if there is one, why aren't the docs referencing it? 
I don't understand how the same filters for events aren't applied to MQTT when I'm using `required_zones` but anyway...

**Reply by hawkeye217** [reply]
From https://docs.frigate.video/integrations/mqtt#frigateevents:

> Message published for each changed event. The first message is published when the tracked object is no longer marked as a false_positive.

As the docs state, every time something changes, it's published to this topic. Any kind of filtering must be done by the consumer.

The [Home Assistant notifications guide](https://docs.frigate.video/guides/ha_notifications) give more:

> It is generally recommended to trigger notifications based on the frigate/events mqtt topic. This provides the event_id needed to fetch [thumbnails/snapshots/clips](https://docs.frigate.video/integrations/home-assistant#notification-api) and other useful information to customize when and where you want to receive alerts. The data is published in the form of a change feed, which means you can reference the "previous state" of the object in the before section and the "current state" of the object in the after section.

If you have any other questions on how it works, feel free to ask, we're happy to help.

**Reply by Morphy99** [reply]
My bad, I'd forgotten that MQTT required_zones is only for snapshots. So if I wanted to filter notifications to zones I can use this?
`{{ 'path_close' in trigger.payload_json['after']['entered_zones'] }}`

Does this apply the object filters for area size which are defined in the zones in per camera config?

I have notes to say the 'payload: new' messages don't include entered zones however which would mean a slight delay and having to use 'payload: update' messages?

Basically, I just want the fastest notifications for events with the same filters I have set to avoid false positives.

I've come across Nick's [guide](https://github.com/blakeblackshear/frigate/discussions/11554) at using the new review notifications which I'm going to give a go. One question about that, does the notification trigger at the end of the event?

**Reply by hawkeye217** [reply]
> My bad, I'd forgotten that MQTT required_zones is only for snapshots. So if I wanted to filter notifications to zones I can use this? `{{ 'path_close' in trigger.payload_json['after']['entered_zones'] }}`
> 

Yep, that looks right.

> Does this apply the object filters for area size which are defined in the zones in per camera config?
> 
> I have notes to say the 'payload: new' messages don't include entered zones however which would mean a slight delay and having to use 'payload: update' messages?
> 
> Basically, I just want the fastest notifications for events with the same filters I have set to avoid false positives.
> 
> I've come across Nick's [guide](https://github.com/blakeblackshear/frigate/discussions/11554) at using the new review notifications which I'm going to give a go. One question about that, does the notification trigger at the end of the event?

The `review` topic is probably the better source for notifications in 0.14. As with the `events` topic, you'll see `end` as the message type when the review time period ends.
